src/elements-experimental/timetable-row/timetable-row.visual.spec.ts

```diff
@@ -85,7 +85,7 @@ describe(`sbb-timetable-row`, () => {
       ?loading-trip=${args.loadingTrip}
       ?a11y-footpath=${args.a11yFootpath}
       ?loading-price=${args.loadingPrice}
-      now=${(args.now ?? new Date('2022-12-01T12:11:00').valueOf()) / 1000}
+      now=${args.now ?? new Date('2022-12-01T12:11:00')}
```

Comment:
```suggestion
      .now=${args.now ?? new Date('2022-12-01T12:11:00')}
```

Reply:
Thanks! I was trying to figure out where I missed it. 😅